lib/dk_buttons_and_leds/Kconfig

```diff
@@ -22,4 +22,8 @@ module = DK_LIBRARY
 module-str = DK library
 source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
 
+config DK_LIBRARY_SHELL
+	bool "Enable shell to control DK_LIBRARY and allow to simulate button press"
```

Comment:
please avoid `Enable` in prompts.

Also remember that the people who will see the line are those using menuconfig of similar, therefore we should avoid long texts as prompt, instead the `help` keyword should be used for detailed description.

The bonus on that is that the help text will automatically be rendered into NCS documentation.

Like this:
```
config DK_LIBRARY_SHELL
	bool "<Short and as descriptive as possible title of setting>"
Comment
	depends on SHELL
	help
	  <Long and detailed description of this setting>
```

Reply:
refactored Kconfig entry as suggested